### ISSUE TITLE
Extract the bounding box after downloading datasets

### ISSUE THREAD

**Issue by davidgamez** [MobilityData/mobility-feed-api#230]
### Describe the problem

As part of the Datasets metadata, we would like to include the bounding box information. This issue implements extraction and update of the bounding box of the dataset in the MobilityData Database.

Depends on #225

### Proposed solution

Using the configuration created in #225, create a function that extracts the bounding box and updates the MobilityData Database.

### Alternatives you've considered

Include the script as part of the implementation of #228. This implementation was discarded as the execution of the validator, and the extraction of the bounding box follows a different implementation pattern.

**Comment by cka-y** [comment]
I believe this is a duplicate of #173
